draft-jasdips-regext-rdap-rpki.md

```diff
@@ -74,72 +75,77 @@ Furthermore, correlating registered RPKI data with registered IP networks and au
 access to the latter's contact information through RDAP entity objects, which should aid troubleshooting.
 
 In addition to troubleshooting, serving RPKI meta-data over RDAP offers a convenience to network operators
-through a simple lookup mechanism. As is demonstrated in the [@?RDAP-GUIDE](https://rdap.rcode3.com/misc/uses.html),
-constructing custom RDAP scripts is relatively easy and beneficial to network operators for the purposes of
-reporting. Systems such as [@?JDR](https://blog.nlnetlabs.nl/introducing-jdr/) have shown the utility of such
-an approach. 
+through a simple lookup mechanism. As is demonstrated in [@RDAP-GUIDE], constructing custom RDAP scripts is
+relatively easy and beneficial to network operators for the purposes of reporting. Systems such as [@JDR] have shown the
```

Comment:
I do not think JDR consumed RDAP. It was a piece of Julia code that parsed the RPKI objects. Will add a suggestion.

Reply:
It didn't do RDAP, but it did allow getting the information via the web. The point here is that sometimes it is beneficial to get the information via another method.